arch/x86/um/asm/elf.h

```diff
@@ -187,6 +187,7 @@ do {								\
 struct linux_binprm;
```

Comment:
In commit message, syscall -> syscalls

And arch finalize exec should be also mentioned